03.bowling/y-bowling.rb

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+score = ARGV[0]
+split_score = score.split(',')
+shots = []
+
+split_score.each do |s|
+  if shots.length < 18
+    if s == 'X'
+      shots << 10
+      shots << 0
+    else
+      shots << s.to_i
+    end
+  else
+    shots << if s == 'X'
+               10
+             else
+               s.to_i
+             end
```

Comment:
```suggestion
    shots << (s == 'X' ? 10 : s.to_i)
```

三項演算子の方がすっきりしそうです

さらにいうと、ロジック的にはここまで短くできるかも？

```ruby
split_score.each do |s|
  shots << (s == 'X' ? 10 : s.to_i)
  shots << 0 if shots.length < 18 && s == 'X'
end
```

Reply:
３項演算子は理解できていたつもりになっていましたが、このように書けるのは正直思いもしませんでした。
こういったテクニックを教えていただけるのはとても良い経験となります😁